cli/tools/test/mod.rs

```diff
@@ -547,6 +573,17 @@ pub async fn run_tests_for_worker(
     used_only,
   }))?;
   let mut had_uncaught_error = false;
+  let stats = worker.js_runtime.runtime_activity_stats_factory();
+  let ops = worker.js_runtime.op_names();
+  let op_id_host_recv_message = ops
+    .iter()
+    .position(|op| *op == "op_host_recv_message")
+    .unwrap();
+  let op_id_host_recv_ctrl = ops
+    .iter()
+    .position(|op| *op == "op_host_recv_ctrl")
+    .unwrap();
```

Comment:
Let's add a comment here, describing why these two ops are special - maybe to make the dependency explicit we could import their op declarations from `deno_runtime` crate and use `<name>::DECL.name`?